src/tunnel.cc

```diff
@@ -858,7 +858,7 @@ TunnelStateData::copyRead(Connection &from, IOCB *completion)
     int bw = from.bytesWanted(1, SQUID_TCP_SO_RCVBUF);
     // XXX: Delay pools must not delay client-to-Squid traffic (i.e. when
     // from.readPendingFunc is tunnelDelayedClientRead()).
-    // XXX: Bug #4913: Use DeferredRead instead.
+    // XXX: Bug #4913: for delay pools, use delayDead() API instead.
```

Comment:
```suggestion
    // XXX: Bug #4913: for delay pools, use delayRead() API instead.
```

Reply:
> Your line is the same as mine

GitHub would not highlight the difference if there was none. Your line misspells `Read` as `Dead`.

Reply:
Sorry, missed it. Done.